docs/omero_figure_scripting.rst

```diff
@@ -13,44 +13,45 @@ Setup
 
 -  Install the OMERO.figure web app as described at https://pypi.org/project/omero-figure/
 -  Upload the script :download:`Split_View_Figure.py <../scripts/Split_View_Figure.py>` to the OMERO scripting service
+-  For the :ref:`Shapes heatmap from OMERO.table<omeroTableData>` example, you'll need to draw ROIs on an Image and create an OMERO.table on the
+   Image with a ``Shape`` column with corresponding Shape IDs, as described below.
```

Comment:
I would remove ``below`` - it just confused me, as there is a link in the first part of the sentence, but the ``below`` is not a link, so was scrolling through the doc to find "below". Simplest would be to say "inside the example", but better would be to add the same link you have there already to the word "below"

Reply:
I'd prefer not to add 2 links to the same place with different text as it looks like they're both different (and you might need to click both). So I'll say "in the example".